### PR TITLE
Platform specific configuration and Environment variable expansion

### PR DESCRIPTION
**Pull request by Starli0n**
Hi,

In this PR, there is two new features:

*   Platform specific configuration in User folder
    (Platform settings take precedence on generic settings)
*   Environment variable expansion
    (Replace environment variable in path set in the configuration files)

Thank you

### CODE REVIEW

File: file_diffs.py [hunk 1]
```diff
@@ -51,8 +51,17 @@ def on_done(index):
 
 
 class FileDiffCommand(sublime_plugin.TextCommand):
-    def settings(self):
-        return sublime.load_settings('FileDiffs.sublime-settings')
+    def get_setting(self, key, default=None):
+        print('FileDiffs.sublime-settings')
```
Comment (colinta) [inline file_diffs.py]: Small thing, but could you remove this `print`?

**Comment by colinta** [comment]
Looks great!  Just gonna wait on removing the debug `print()` statement, then it's good to merge.

File: file_diffs.py (hunk 1 above)
Comment (Starli0n) [inline file_diffs.py]: Sure, it was for a debug purpose.
I forgot to remove it. My bad.

**Comment by Starli0n** [comment]
Done :-)

**Comment by colinta** [comment]
Thanks!  I pushed this up as 2.8.0.  It just now occurred to me that we should mention this feature in the README.

**Comment by Starli0n** [comment]
True! I created another PR #78 for that.

**Pull request merged**